lab1/lab1/lab1.cpp

```diff
@@ -0,0 +1,174 @@
+﻿#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <climits>
+#include <ctime>
+#include <cstdlib>
+#include <locale>  
+
+struct Sequence {
+    int value = INT_MIN;
+    std::ifstream* file;
```

Comment:
Рекомендую всегда инициализировать указатели.
```suggestion
    std::ifstream* file = nullptr;
```